src/chordDetail.js

```diff
@@ -103,7 +106,8 @@ class ChordDetail extends Component {
                                 this.setState({major: e.target.value});
                             }}
                             type='boolean' 
-                            value={this.state.major}>
+                            value={this.state.major}
```

Comment:
Super close here! `Boolean` is a `data` type but not an `input` type. In this case, you'd want to use a specific input type that can give the user multiple options, such as a drop-down, radio buttons, or a checkbox. Ref: [HTML Data Types](https://www.w3schools.com/html/html_form_input_types.asp)